js_modules/dagit/packages/core/src/pipelines/PipelineRunsRoot.tsx

```diff
@@ -79,6 +83,8 @@ export const PipelineRunsRoot: React.FC<Props> = (props) => {
     },
   });
 
+  const refreshState = useQueryRefreshAtInterval(queryResult, FIFTEEN_SECONDS);
```

Comment:
Instead of passing this `refreshState` around, should we just have this set it on a context that the countdown reads?

Reply:
Ahh that'd be super cool - I think it's pretty likely that the "closest" `useQueryRefreshAtInterval` is the one you want to display state for. I'll try making this change and see how it looks!